applications/serial_lte_modem/doc/slm_description.rst

```diff
@@ -0,0 +1,499 @@
+
+1. Retrieve the version of the serial LTE modem application.
+
+   .. parsed-literal::
+      :class: highlight
+
+      **AT#XSLMVER**
+      #XSLMVER: 1.2
+      OK
+
+#. Read the current baud rate.
+
+   .. parsed-literal::
+      :class: highlight
+
+      **AT#XSLMUART?**
+      #SLMUART: 115200
+      OK
+
+   You can change the used baud rate with the corresponding set command, but note that LTE Link Monitor requires 115200 bps for communication.
+
+#. Retrieve a list of all supported proprietary AT commands.
+
+   .. parsed-literal::
+      :class: highlight
+
+      **AT#XCLAC**
+      AT#XSLMVER
+      AT#XSLEEP
+      AT#XCLAC
+      AT#XSOCKET
+      AT#XSOCKETOPT
+      AT#XBIND
+      *[...]*
+      OK
+
+#. Check the supported values for the sleep command, then put the board in sleep mode.
+
+   .. parsed-literal::
+      :class: highlight
+
+      **AT#XSLEEP=?**
+      #XSLEEP: (0, 1)
+      OK
+
+      **AT#XSLEEP=1**
+
+   Reset the board to exit sleep mode.
+   If you are testing with an external MCU and :option:`SLM_GPIO_WAKEUP` is enabled, you can wake up by GPIO as well.
+
+
+
+Testing TCP/IP AT commands
+--------------------------
+
+The following sections show how to test the functionality provided by the :ref:`SLM_AT_TCPIP`.
+
+TCP client
+~~~~~~~~~~
+
+1. Establish and test a TCP connection:
+
+   a. Check the available values for the XSOCKET command.
+
+      .. parsed-literal::
+        :class: highlight
+
+        **AT#XSOCKET=?**
+        #XSOCKET: (0, 1), (1, 2), (0, 1), <sec-tag>
+        OK
+
+   #. Open a TCP socket, read information (handle, protocol, and role) about the open socket, and set the receive time-out of the open socket to 30 seconds.
+
+      .. parsed-literal::
+        :class: highlight
+
+        **AT#XSOCKET=1,1,0**
+        #XSOCKET: 2, 1, 0, 6
+        OK
+
+        **AT#XSOCKET?**
+        #XSOCKET: 2, 6, 0
+        OK
+
+        **AT#XSOCKETOPT=1,20,30**
+        OK
+
+   #. Connect to a TCP server on a specified port.
+      Replace ``example.com`` with the host name or IPv4 address of a TCP server and ``1234`` with the corresponding port.
+      Then read the connection status.
+      ``1`` indicates that the connection is established.
+
+      .. parsed-literal::
+        :class: highlight
+
+        **AT#XCONNECT="example.com",1234**
+        #XCONNECT: 1
+        OK
+
+        **AT#XCONNECT?**
+        #XCONNECT: 1
+        OK
+
+   #. Send plain text data to the TCP server, then retrieve it from the TCP server.
+
+      .. parsed-literal::
+        :class: highlight
+
+        **AT#XSEND=1,"Test TCP"**
+        #XSEND: 8
+        OK
+
+        **AT#XRECV**
+        PONG: b'Test TCP'
+        #XRECV: 1, 17
+        OK
+
+   #. Send hexadecimal data to the TCP server, then retrieve it from the TCP server.
+
+      .. parsed-literal::
+        :class: highlight
+
+        **AT#XSEND=0,"DEADBEEF"**
```

Comment:
Does this work after sending plain text data, or would you need to close the socket and open a new one?

Reply:
No, socket is still opened unless it is closed by "AT#XSOCKET=0"